src/Renderer/PokemonRenderer.php

```diff
@@ -74,6 +74,17 @@ public function render(
                 $attacksCollection,
                 $this->translations
             ),
+            'hasShadow' => $pokemon->hasShadow(),
```

Comment:
- [ ] this new fields must be added in the `public/Pokedex.json` OpenAPI Specification.